### ISSUE TITLE
About youtube/281.

### ISSUE THREAD

**Issue by githuboflk** [SCLBD/DeepfakeBench#86]
I found error load image in replicating LSDA. Error from No faces in frame 300 / MNT/sda1 / video/FaceForensics++ original_sequences/youtube/c23 / videos / 281. Mp4. I re-used the data processing code you provided for face cropping. But still faces cannot be discovered from 281.

**Comment by 1ggw1** [comment]
The same question

**Comment by 1ggw1** [comment]
Did you solve it, bro?

**Comment by githuboflk** [comment]
> Did you solve it, bro?

No. I downloaded the data processed by the author. But that still doesn't include youtube/281. And my reproduction results are much worse than in the paper.

**Comment by wang-qichang** [comment]
The same question
